static/stable/stage/modules/fed-modules.json

```diff
@@ -895,4 +895,21 @@
           }
       ]
   }
+  "quay-ui-plugin": {
```

Comment:
I think you are missing a `,` before the new entry.

Reply:
thanks. Fixed it